src/api/schemas.ts

```diff
@@ -66,7 +66,7 @@ export const AddressesParam = Type.Array(AddressParam, {
   ],
 });
 
-export const Brc20TickerParam = Type.String();
+export const Brc20TickerParam = Type.String(); // todo: validate length and add second prefix parameter
```

Comment:
Validating length here won't work because it's not the `.length` that we want to constrain but the string's size in bytes. Typebox doesn't have a validator for byte length.